desktop-app/src/common/deviceList.ts

```diff
@@ -156,7 +192,7 @@ export const defaultDevices: Device[] = [
     isMobileCapable: true,
   },
   {
-    id: '10011',
+    id: '10015',
```

Comment:
Please don't modify the ids of existing devices, append the new devices to the end of the apple products in this list.

Reply:
I made the changes